charts/activemq-artemis/values.yaml

```diff
@@ -11,8 +11,12 @@ imageTag: latest
 
 ## Create a database user
 ##
-artemisUser: artemis
-artemisPassword: simetraehcapa
+artemis:
+  useDefaultUser: true
+  defaultUser:
+    artemisUser: artemis
+    artemisPassword: simetraehcapa
+  secretref: secretref
```

Comment:
What kind of secretRef? Please provide documentation in the form of comments around this code and code block. And maybe something like `customSecretRef`?